docs/user/kubevirt/role.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: tenantcluster-installer
+  namespace: <tenantcluster-namespace>
+rules:
+- apiGroups: [""]
```

Comment:
don't we need RBAC to create service accounts as well? at least when cloud-credentials-controller will support mint mode?

Reply:
Will be added when the support will be added